snow/consensus/snowstorm/test_tx.go

```diff
@@ -20,9 +19,6 @@ type TestTx struct {
 	DependenciesV    []Tx
```

Comment:
Are we removing the remaining snowstorm later?

Reply:
We could likely simplify the interface - but we won't be able to fully delete it until we no long allow syncing the DAG

Reply:
> syncing the DAG

Ah, right :)